### PR TITLE
Added POST /consents and POST/PUT /consentRequests call and bumped version

### PR DESCRIPTION
**Pull request by kleyow**
To be merged after #74

### CODE REVIEW

File: src/index.d.ts [hunk 1]
```diff
@@ -4,6 +4,10 @@ declare namespace SDKStandardComponents {
...
 
+    type TAuthScope = 'accounts.getBalance' | 'accounts.transfer';
```
Comment (lewisdaly) [inline src/index.d.ts:7]: I think this can be just a plain string, since it could end up being DFSP specific.

File: src/index.d.ts [hunk 2]
```diff
@@ -57,6 +61,23 @@ declare namespace SDKStandardComponents {
         extensionList?: TExtensionList
     }
 
+    type TCredential = {
+        id: string | null;
+        credentialType: 'FIDO';
+        // TODO: Double check if enum is ACTIVE or VERIFIED.
```
Comment (lewisdaly) [inline src/index.d.ts:67]: TODO - are you planning to do in this PR?

Comment (kleyow) [reply src/index.d.ts]: yes. my mistake. it's supposed to be `ACTIVE`. comment removed.

File: src/index.d.ts (hunk 1 above)
Comment (kleyow) [reply src/index.d.ts]: done.

File: src/index.d.ts [hunk 3]
```diff
@@ -57,6 +59,22 @@ declare namespace SDKStandardComponents {
         extensionList?: TExtensionList
     }
 
+    type TCredential = {
+        id: string | null;
+        credentialType: 'FIDO';
+        status: 'PENDING' | 'ACTIVE';
+        challenge: {
+            payload: ArrayBuffer;
```
Comment (eoln) [inline src/index.d.ts:67]: same questions like for #74 for the usage of `ArrayBuffer` instead of `string`

File: src/index.d.ts [hunk 4]
```diff
@@ -57,6 +59,22 @@ declare namespace SDKStandardComponents {
         extensionList?: TExtensionList
     }
 
+    type TCredential = {
+        id: string | null;
```
Comment (eoln) [inline src/index.d.ts:63]:
```suggestion
        id: string;
```
when `id` could be `null`

Comment (kleyow) [reply src/index.d.ts:63]: It can't. fixed.

**Pull request merged**